virt-v2v/cold/entrypoint.go

```diff
@@ -195,33 +195,38 @@ func LinkDisks(diskKind string, num int) (err error) {
 	return
 }
 
-func executeVirtV2v(args []string) (err error) {
-	virtV2vCmd := exec.Command(args[0], args[1:]...)
-	r, w := io.Pipe()
-	virtV2vCmd.Stdout = w
-	virtV2vCmd.Stderr = w
-
-	fmt.Println("exec ", virtV2vCmd)
-	if err = virtV2vCmd.Start(); err != nil {
-		fmt.Printf("Error executing command: %v\n", err)
-		return
+func executeVirtV2v(args []string) error {
+	v2vCmd := exec.Command(args[0], args[1:]...)
+	monitorCmd := exec.Command("/usr/local/bin/virt-v2v-monitor")
+	monitorCmd.Stdout = os.Stdout
+	monitorCmd.Stderr = os.Stderr
+
+	var writer *io.PipeWriter
+	monitorCmd.Stdin, writer = io.Pipe()
+	v2vCmd.Stdout = writer
+	v2vCmd.Stderr = writer
+	defer writer.Close()
+
+	if err := monitorCmd.Start(); err != nil {
+		fmt.Printf("Error executing monitor command: %v\n", err)
```

Comment:
We can use `"log/slog"`

Reply:
@bennyz do you mind posting this change is separately?

Reply:
resolving, this can be posted separately